frontend/packages/client/src/pages/Proposal.js

```diff
@@ -127,12 +125,6 @@ export default function ProposalPage() {
         }
       : {};
 
-  useEffect(() => {
-    if (modalContext.isOpen && user?.addr && !voteError && !cancelProposal) {
-      modalContext.closeModal();
-    }
-  }, [modalContext, user, voteError, cancelProposal]);
-
```

Comment:
This not used anymore because now we show the wallet connect modal instead of a modal with the wallet connect button